src/bundle/Controller/ContentEditController.php

```diff
@@ -65,6 +65,10 @@ public function __construct(
      * @param int|null $locationId
      *
      * @return ContentEditView|Response
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
```

Comment:
This is not related to this ticket. Please create new PR with this change.

Reply:
I'd also advise creating new JIRA ticket for this change.

Reply:
Describing how QA can test this change wouldn't harm either. Ah Andrew... I was expecting more from you :/

Reply:
If we really throw exceptions we don't catch, I guess we should document it. /cc: @DominikaK

Reply:
![i-dont-believe-you](https://user-images.githubusercontent.com/1736658/37451273-697e8644-2831-11e8-97e8-30913260b890.gif)

Reply:
And I don't like the font.